v1/commandfilereturn_results.go

```diff
@@ -0,0 +1,22 @@
+/*
+ * JumpCloud APIs
+ *
+ * V1 & V2 versions of JumpCloud's API. The previous version of JumpCloud's API. This set of endpoints allows JumpCloud customers to manage commands, systems, & system users.
```

Comment:
Curious where this line comes from. It's worded a little odd. and seems to be in a lot of places.